hexrdgui/main_window.py

```diff
@@ -1779,4 +1779,8 @@ def dropEvent(self, paths):
             # extension is not in known list
             self.open_image_files(selected_files=paths)
         except Exception as e:
-            print(f'Error loading images: {e}')
+            error_message = (
+                'Unable to guess file type (state, instrument, materials, or '
+                'image). Please use File menu to load.'
+            )
```

Comment:
```suggestion
            )
            print(error_message)
```
Can you also print this out, Brianna?

Reply:
Oh actually, scratch this. I think there's something better to do.